src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -714,7 +714,7 @@ private bool InferNamedPipesInformation()
                         return false;
                     }
 
-                    if (tokensByBackSlash[4].StartsWith(NamedPipeInstanceNameHeader))
+                    if (tokensByBackSlash[4].StartsWith(NamedPipeInstanceNameHeader, StringComparison.OrdinalIgnoreCase))
```

Comment:
No need to ignore case.
```suggestion
                    if (tokensByBackSlash[4].StartsWith(NamedPipeInstanceNameHeader, StringComparison.Ordinal))
```

Reply:
The StringComparison.Ordinal is now used instead.